main.go

```diff
@@ -47,7 +47,7 @@ func main() {
 	http.Handle("/metrics", promhttp.Handler())
 	go http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
 
-	ac := controller.NewALBController(&aws.Config{MaxRetries: aws.Int(5)}, conf)
+	ac := controller.NewALBController(&aws.Config{MaxRetries: aws.Int(20)}, conf)
```

Comment:
Should we make this configurable by exposing it as an annotation on controller?

Defaulting to 20.